components/Scholar.js

```diff
@@ -1,26 +1,27 @@
 import Image from 'next/image'
+import scholar_one from '../public/scholar_one.jpg'
```

Comment:
What benefit does this import is bringing?

Reply:
![image](https://user-images.githubusercontent.com/44665645/148829952-d3c47da5-24ea-45e4-b65c-49a30b6aa030.png)

"Required, except for statically imported images."
Considering this, we don't have to give width and height when we import it.